app/src/test/java/org/astraea/partitioner/nodeLoadMetric/DependencyPartitionTest.java

```diff
@@ -46,7 +46,8 @@ void testDependencyPartitioner() {
         ps.entrySet().stream()
             .collect(Collectors.toMap(e -> e.getKey().toString(), Map.Entry::getValue));
     try (var producer = Producer.builder().configs(props).build()) {
-      var dependencyClient = SmoothWeightPartitioner.beginDependency(producer.kafkaProducer());
+      var dependencyClient = SmoothWeightPartitioner.dependencyClient(producer.kafkaProducer());
```

Comment:
Did you check the partition in this test (`testDependencyPartitioner`)?

Reply:
sure 
in `if (i > 0) Assertions.assertEquals(targetPartition, metadata.partition());`